src/test/store/profile-view.test.js

```diff
@@ -1344,6 +1370,14 @@ describe('snapshots of selectors/profile', function() {
       'Merge: C',
     ]);
   });
+  it('matches the last stored run of selectedThreadSelector.getTabFilteredThread', function() {
```

Comment:
Optional: This will get the job done quick, but it might be better to write non-snapshot tests.